### DISCUSSION TITLE
Will there be advantage of creating CephCluster on existing NFS storage?

### DISCUSSION THREAD

**Discussion by Chakravarti-Baratam** [rook/rook#9828] [General]
Hi,

Will there be advantage of creating CephCluster on existing NFS storage?

In my k8s clusters, nodes are not having enough storage to run bare metal CephCluster. As I have external NFS, I thought of creating CephCluster using external NFS storage. 

Then I am wondering, what is the benefit of this set up. Please help me.

Thanks,
Chakri.

**Comment by travisn** [comment]
The CephCluster [requires block storage](https://rook.github.io/docs/rook/v1.8/quickstart.html#prerequisites) on which to be configured, so it won't be possible to deploy ceph on top of nfs.
